cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/ArtifactStore.java

```diff
@@ -189,9 +192,10 @@
   private final DatasetFramework datasetFramework;
   private final Transactional transactional;
   private final Impersonator impersonator;
+  private final Set<String> excludedRequirements;
```

Comment:
Call the field `requirementsBlackList` to be consistent with the config name

Reply:
Done.